src/index.css

```diff
@@ -7,6 +7,10 @@ body {
   -moz-osx-font-smoothing: grayscale;
 }
 
+div root{
+  font-size: 13px;  
+}
```

Comment:
I don't _think_ that this will actually apply to anything.
This is looking for a `root` tag which is a child of a `div` tag.

But I don't think a `root` tag exists, and if it did, it would the parent of the entire page so wouldn't be found within a div.

I think what we want it either of 
```css
:root {
    /* pseudo-class selector for the root element of the page */
}

html {
    /* selector for the html tag which is the root of our document. */
}
```